lib/Controller/Widget.php

```diff
@@ -349,7 +349,7 @@ public function editWidgetForm(Request $request, Response $response, $id)
             'commonProperties' => [
                 'name' => $widget->getOptionValue('name', null),
                 'enableStat' => $widget->getOptionValue('enableStat', null),
-                'isRepeatData' => $widget->getOptionValue('isRepeatData', null),
+                'isRepeatData' => $widget->getOptionValue('isRepeatData', true),
```

Comment:
This was null before because it is not always required (we don't use it on widgets without data for example)